heron/api/src/java/com/twitter/heron/api/bolt/WindowedBoltExecutor.java

```diff
@@ -237,9 +237,23 @@ protected void validate(Map<String, Object> topoConf, Count windowLengthCount, L
     // validate
     validate(topoConf, windowLengthCount, windowLengthDurationMs, slidingIntervalCount,
```

Comment:
If a custom trigger or evictor, validate should not be called.  Arguments would be directly passed into the custom evictor and trigger when they are intialized